src/main/java/core/basesyntax/db/FruitShopDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface FruitShopDao {
+    int storageSize();
```

Comment:
The method name 'storageSize' is not very descriptive of what it does. Consider renaming it to something more specific, like 'getStorageSize', to follow the Java naming conventions for methods that return a value.